sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthcareEntitiesAction.cs

```diff
@@ -49,7 +49,7 @@ public AnalyzeHealthcareEntitiesAction(AnalyzeHealthcareEntitiesOptions options)
         /// </para>
         /// </summary>
         /// <remarks>
-        /// This property only applies for <see cref="TextAnalyticsClientOptions.ServiceVersion.V3_1"/>, <see cref="TextAnalyticsClientOptions.ServiceVersion.V2022_05_01"/>, and up.
+        /// This property only applies for <see cref="TextAnalyticsClientOptions.ServiceVersion.V3_1"/>, <see cref="TextAnalyticsClientOptions.ServiceVersion.V2022_05_01"/>, and newer.
```

Comment:
AnalyzeHealthcareEntitiesAction wasn't supported until Language API, should the disclaimer should apply for the entire action type?

Reply:
Good catch. I copied and pasted everywhere, but this would be misleading.